fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -996,6 +997,10 @@ private void handleShowCreateDb() throws AnalysisException {
         if (!Strings.isNullOrEmpty(db.getLocation())) {
             createSqlBuilder.append("\nPROPERTIES (\"location\" = \"").append(db.getLocation()).append("\")");
         }
+        if (RunMode.getCurrentRunMode() == RunMode.SHARED_DATA && !Strings.isNullOrEmpty(db.getStorageVolumeId())) {
```

Comment:
does it mean that in shared nothing mode, create db/table statement is not allowed to specify 'storage volume' property?

I would say, we should respect what the db storageVolumeId is no matter it is a shared data or shared nothing mode. If the db is not allowed to have a storage volume property, just block it when creating/altering sql.

Reply:
I will block creating db using storage volume in shared_nothing mode in next PR.

Reply:
then you don't have to check runMode here, just need to check if the db's storageVolumeId is empty or not, right?

Reply:
yes. fixed it.